mpi-proxy-split/dbg_logger/mpi_logger_utils.h

```diff
@@ -0,0 +1,142 @@
+#ifndef _MPI_LOGGER_UTILS_H
+#define _MPI_LOGGER_UTILS_H
+
+#include <mpi.h>
+
+#define EXTERNC extern "C"
+#define ENABLE_LOGGER_PRINT true
```

Comment:
Please change `true` to `1`, above.  That's more conventional for the C preprocessor.

Actually, `#if ENABLE_LOGGER_PRINT` will simply check if this is t to `0` or any non-zero value.  Probably, even `false` counts as a true value.  The preprocessor works with any language -- not just C++.

An alternate from is `#ifdef ENABLE_LOGGER_PRINT`.  In that form, it simply checks if you did even `#define ENABLE_LOGGER_PRINT` (with no value for the preprocessor variable.  In that case, you comment it out if you don't want it, and even `0` counts as a _defined_ value.